README.md

```diff
@@ -21,7 +21,24 @@ Support for:
   - Zarr v2
 
 ## Installation
+> **Note** We recommend installing acquire-imaging in a clean python virtual environment 
+> using an environment manager such as conda. You may install [miniconda](https://docs.conda.io/en/latest/miniconda.html)
+> as a minimal installer and environment manager for conda. 
 
+Create a new virtual environment called "acquire" and install Python 3.10 using the following:
+```bash
+conda create --name acquire python=3.10
+conda activate acquire
+```
+
+[Rust](https://www.rust-lang.org/learn/get-started) must be installed as a prerequisite for acquire-imaging installation. 
+> **Note** If you encounter a permission error with MacOS when installing rust, try the following:
+> ```bash
+> curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | \
+> bash -s -- -y --no-modify-path
+> ```
```

Comment:
Installing rust is something we'd like users to be able to avoid. Based on the discussion in #55, it looks like you ran into this problem because we weren't providing binaries on pypi (that's resolved now).

When the right wheel isn't available, `pip` will try to build from source, but that's much more involved. Building from source fits the developer focused docs more.